docs/parameters.yaml

```diff
@@ -2496,6 +2496,36 @@ type: duration
 default: 360h
 components: ["origin", "cache", "director", "registry"]
 ---
+name: Monitoring.LabelLimit
+description: |+
+  The maximum number of labels that can be attached to a single metric. 0 means no limit.
+type: int
+default: 64
+components: ["origin", "cache", "director", "registry"]
+---
+name: Monitoring.LabelNameLengthLimit
+description: |+
+  The maximum length of a label name. 0 means no limit. The default value is 128 bytes, which allows for up to 32 characters.
```

Comment:
It'd be helpful to include a quick example of what a single "metric" looks like, and how the "label" and "value" keys work in it. Maybe a triple-backtick formatted JSON would work to show that here?

It'd also be useful to provide some insight about the consequences of picking a "bad" value here. You can say something about how these numbers may directly affect the memory consumption of our services, and caution the user from overriding the defaults.